chrome/locale/en-US/zotero/zotero.properties

```diff
@@ -293,6 +293,9 @@ pane.item.related.count.singular = %S related:
 pane.item.related.count.plural  = %S related:
 pane.item.parentItem			= Parent Item:
 
+drag.invalid.title 				= Invalid Item
+drag.publications.onlyTopLevel	= Only top-level Zotero items are allowed in My Publications
```

Comment:
They are, but the problem is just that notes and attachments are Zotero items too.